Modix.Services/Messages/MessageService.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Threading.Tasks;
+using Discord;
+using Discord.WebSocket;
+using Modix.Data.Repositories;
+using Modix.Services.Core;
+
+namespace Modix.Services.Messages
```

Comment:
I vote this all belongs in `Modix.Services.Core` alongside `UserService`, `ChannelService`, `RoleService`, etc.

Reply:
Probably, if we're really committed to lumping all of that stuff into Core. I've always felt that the Core category just makes it harder to find stuff, especially since the only real relation between its inhabitants is that they deal with Discord entities.

Reply:
Hmmm..... I suppose there's something to that.

The closest thing I've had to a criteria for when something goes in Core or not is whether or not it could someday be pulled out into a plugin. Or, maybe more generally, whether other modules rely on it.

Reply:
As a side-note, I actually have an `IMessageService` in a local branch right now, as part of some larger testing and refactoring, and was planning on putting it in `Core` because the method I need on it is `TrackMessageAsync()`, matching the pattern of `TrackUserAsync()`, and `TrackChannelAsync()` and `TrackRoleAsync()` on those other Core services.

Reply:
Fiiiiiiiiiiine :p